src/Media/MediaUrlRepository.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Media;
+
+final class MediaUrlRepository
+{
+    private array $mapping;
+
+    public function __construct(array $mapping)
+    {
+        $this->mapping = $mapping;
+    }
+
+    public function updateUrl(string $oldUrl): string
```

Comment:
This is not doing a real update, it returns an updated URL. So I think a better name is `getUpdatedUrl`